satellite/metainfo/loop.go

```diff
@@ -37,6 +37,8 @@ type ScopedPath struct {
 	ProjectID       uuid.UUID
 	ProjectIDString string
 	BucketName      string
+	Segment         string
```

Comment:
Changes in this file are copied from #3578 and should be removed when it will be merged.